learning_assistant/models.py

```diff
@@ -1,3 +1,39 @@
 """
 Database models for learning_assistant.
 """
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+
+class CoursePrompt(TimeStampedModel):
+    """
+    This model represents a mapping between a particular course ID and a text prompt associated with the course ID.
+
+    .. no_pii: This model has no PII.
+    """
+
+    # course ID with which the text prompt is associated with
+    course_id = models.CharField(max_length=255, db_index=True, unique=True)
```

Comment:
I think we should use a `CourseKeyField` here, if we can. The [Opaque Keys (Locators)](https://github.com/openedx/edx-platform/wiki/Opaque-Keys-(Locators)) wiki has a good summary of the problems this field solves. What do you think?

If we ever need to manipulate or extract information from the course key (e.g. get the associated organization), using a course key field will make that a lot easier, for example.

Reply:
I'll take a look - I know we used a CharField for exams here: https://github.com/edx/edx-exams/blob/c3bdd6c76da2caaa53076798d7e5cfa9cbab9276/edx_exams/apps/core/models.py#L91.

I'm pretty open to either - and I think the option to have more info is better!

Reply:
Interesting! I wonder why.

I do see `opaque_keys` used throughout our various repositories (including IDAs and plugins), so I can't think of a good reason not to.

https://github.com/search?q=org%3Aopenedx%20opaque_keys&type=code